automated/linux/ltp-open-posix/ltp-open-posix.sh

```diff
@@ -63,6 +66,22 @@ parse_ltp_output() {
 
 }
 
+# Parse LTP open posix grp output
+parse_ltp_grp_output() {
+    sed -i.bak '/^PASS\|^FAIL\|^Testing/!d' "${GRP_OUTPUT_FILE}"
+    paste -d " "  - - - < "${GRP_OUTPUT_FILE}" | tee "${GRP_RESULT_FILE}"
+    sed -i 's/\s\s*/ /g' "${GRP_RESULT_FILE}"
+    sed -i 's/Testing //g' "${GRP_RESULT_FILE}"
+    {
+    grep -E "FAIL 0" "${GRP_RESULT_FILE}" | awk '{print $1 " " $2}' | sed 's/PASS/pass/; s/FAIL/fail/; s/CONF/skip/';
+    grep -v "FAIL 0" "${GRP_RESULT_FILE}" | awk '{print $1 " " $4}' | sed 's/PASS/pass/; s/FAIL/fail/; s/CONF/skip/';
```

Comment:
Can we squash these two lines into one?
something like this:
| sed 's/PASS/pass/; s/FAIL/fail/; s/CONF/skip/'|awk '{ if($5 == 1) {print $1 " " $4} else {print $1 " " $2} }'

Reply:
@roxell thanks. Updated the PR to use single line and tested the change. https://lava.collabora.co.uk/scheduler/job/4202810